src/main/java/teammates/storage/sqlentity/UsageStatistics.java

```diff
@@ -0,0 +1,157 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a system usage statistics for a specified period of time.
+ *
+ * <p>Note that "system usage" here is defined as user-facing usages, such as number of entities created
+ * and number of actions, as opposed to system resources such as hardware and network.
+ */
+@Entity
+@Table(name = "UsageStatistics")
+public class UsageStatistics extends BaseEntity {
+    @Id
+    private String id;
+
+    @Column
+    private Instant startTime;
+
+    @Column
+    private int timePeriod;
+    @Column
+    private int numResponses;
+    @Column
+    private int numCourses;
+    @Column
+    private int numStudents;
+    @Column
+    private int numInstructors;
+    @Column
+    private int numAccountRequests;
+    @Column
+    private int numEmails;
+    @Column
+    private int numSubmissions;
+
+    @CreationTimestamp
+    @Column(updatable = false)
+    private Instant createdAt;
+
+    @UpdateTimestamp
+    @Column
+    private Instant updatedAt;
+
+    @Column
+    private Instant deletedAt;
+
+    protected UsageStatistics() {
+        // required by Hibernate
+    }
+
+    private UsageStatistics(
```

Comment:
Let's stick to the builder pattern so that all entities are created in the same way, see the other entities.